apprating/src/main/java/com/mindera/skeletoid/apprating/store/AppRatingStore.kt

```diff
@@ -0,0 +1,63 @@
+package com.mindera.skeletoid.apprating.store
+
+import android.content.Context
+import android.content.SharedPreferences
+import asia.fivejuly.securepreferences.SecurePreferences
+
+class AppRatingStore {
+
+    companion object {
+        private const val SHARED_PREFS_FILE = "google_play_review_preferences"
+
+        private const val HAS_REVIEWED_KEY = "has_reviewed_key"
+        private const val INITIAL_PROMPT_DATE_KEY = "initial_prompt_date_key"
+        private const val PROMPTED_COUNT_KEY = "prompted_count_key"
+        private const val LAST_TIME_PROMPTED_KEY = "last_time_prompted_key"
+    }
+
+    private val sharedPreferences: SharedPreferences
+
+    constructor(context: Context) {
+        this.sharedPreferences = instantiateSharedPreferences(context)
+    }
+
+    constructor(sharedPreferences: SharedPreferences) {
+        this.sharedPreferences = sharedPreferences
+    }
+
+    private fun instantiateSharedPreferences(context: Context): SharedPreferences = SecurePreferences.Builder(context)
```

Comment:
Don't see a need for this to be encrypted. Besides that, the encryption can generate problems, so I'd say we revert this to standard SharedPreferences